### PR TITLE
feat(telephony time conditions): add export/import options

### PR DESCRIPTION
**Pull request by JeremyDec**
## Title of the Pull Requests
Add export/import feature for time conditions

### CODE REVIEW

**Comment by JeremyDec** [comment]
@Jisay using bulk action don't assure to set all conditions correctly

File: client/app/telecom/telephony/service/time-condition/import/telecom-telephony-service-time-condition-import.controller.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+angular.module("managerApp").controller("TelecomTelephonyServiceTimeConditionImportCtrl", class TelecomTelephonyServiceTimeConditionImportCtrl {
+
+    constructor ($translate, $uibModalInstance, Toast, voipTimeConditionConfiguration) {
+        this.$translate = $translate;
+        this.$uibModalInstance = $uibModalInstance;
+        this.Toast = Toast;
+        this.voipTimeConditionConfiguration = voipTimeConditionConfiguration;
+
+        this.$onInit();
```
Comment (antleblanc) [inline client/app/telecom/telephony/service/time-condition/import/telecom-telephony-service-time-condition-import.controller.js]: not necessary

File: client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html [hunk 2]
```diff
@@ -136,9 +136,22 @@ <h1 data-translate="telephony_line_calls_time_condition_title"></h1>
                                    data-on-error="$ctrl.onBulkError">
             </telephony-bulk-action>
 
-            <voip-time-condition-calendar data-voip-time-condition="$ctrl.line.timeCondition"></voip-time-condition-calendar>
+            <button type="button"
+                    class="btn btn-primary"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/timeCondition/telecom-telephony-line-calls-time-condition.html]: Here we have two primary button?

File: client/app/telecom/telephony/alias/configuration/timeCondition/oldPabx/telecom-telephony-alias-configuration-time-condition-old-pabx.html [hunk 3]
```diff
@@ -132,8 +132,23 @@ <h1 data-translate="telephony_alias_configuration_time_condition_title"></h1>
                                    data-on-error="$ctrl.onBulkError">
             </telephony-bulk-action>
 
-            <voip-time-condition-calendar data-voip-time-condition="$ctrl.number.feature.timeCondition"></voip-time-condition-calendar>
-        </div> <!-- CONDITIONS -->
+            <button type="button"
+                    class="btn btn-primary"
```
Comment (antleblanc) [inline client/app/telecom/telephony/alias/configuration/timeCondition/oldPabx/telecom-telephony-alias-configuration-time-condition-old-pabx.html]: Two primary button?

**Pull request merged**